### DISCUSSION TITLE
The software cannot be opened and used smoothly.

### DISCUSSION THREAD

**Discussion by 3539620500** [Avnsx/fansly#94] [Q&A]
Hello, I'm sorry to bother you. I am a novice with limited experience in software operation. I encountered difficulties while using the software. After changing the config and opening the software, the system prompted "Update check failed, due to no internet connection!" However, I have already opened Clash for Windows and tried other VPN software, but none of them can run smoothly. Is there any solution to this problem? I would be very grateful if you could reply.

**Comment by Avnsx** [comment]
Sound like your firewall or some anti virus is blocking it from accessing the internet

**Reply by 3539620500** [reply]
Thank you for your response. Due to some matters, I was unable to respond immediately. As you suggested, I disabled the firewall and proxy service on my computer. After that, I opened the TUN mode in Clash for Windows, and it seems that the program is now able to run. However, it displays "Reading Configuration" for about twenty seconds, then flashes a piece of code and closes the window. What could be causing this issue?
The code appears for a short time, maybe only 0.5 seconds, and I tried to record it with my phone that seemed to be these
File "urllib3\connection.py"， line 358, in connect

File "urllib3\connection.py", Line 186, in _new_conn

uriis63.exceptions.NewConnectiónErrori <urLLi63.connection.HTTPSConnection object af 9x0ooo025FA8327490i4olee blish a new connection:[WinError 10060]由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。During handling of the above exception, another exception occurred:

Traceback (most recent call last):

File "requests ladapters.py", Line 440, in send

File "urlLib3\connectionpooi.py", Line 785, An urlopen

File "urlLib3\util\retry.py",Line 592, in increment,

wiii63.exceptions.NaxRetryError: HTTPSConnectionPoo\Chost='apiv3.fansly.comi

port=443): Max retries exceeded wi/api/v1/account?usernames= (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection objee 0000025FA8323490>: Failed to establish a new connection: [WinError 10060]由于连接方在一段时间后没有正确答复或连没有反应，连接尝试失败·))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):

File "fansly_scraper.py", line 137, in <module>File "requests\sessions.py",line 542, in get File "requests\sessions.py", line 529, in request File "requests\sessions.py", line 645, in send File "requests \adapters.py", line 519, in send

requests.exceptions.ConnectionError: HTTPSConnectionPool(host='apiv3.fansly.com', port=443): Max retries exceeded rl： /api/v1/account?usernames=

**Reply by Avnsx** [reply]
The new error you're getting also means that the program still can't connect to specific api endpoints. Your network or anti-virus programs or whatever are still partially blocking fansly downloaders connections to the internet.

Right now it's blocking the connection to ``/api/v1/account?usernames=``

Enable a VPN and set it to a european country or something, that might help bypassing the blocking of the connections to fansly.
